veracruz-server-test/src/main.rs

```diff
@@ -1286,8 +1298,7 @@ mod tests {
 	// the policy files. so we need to replace it with the private IP of the current instance
         #[cfg(feature = "nitro")]
         {
-            let ip_string = local_ipaddress::get()
-            .expect("Failed to get local ip address");
+            let ip_string = std::env::var("TABASCO_IP_ADDRESS").expect("TABASCO_IP_ADDRESS environment variable not set");
```

Comment:
I think this should be something like `PROXY_ATTESTATION_SERVER_ADDRESS`, given we've moved away from Tabasco as part of the Great Renaming.